sympy/categories/diagram_drawing.py

```diff
@@ -2155,14 +2155,14 @@ def _check_free_space_vertical(dom_i, cod_i, dom_j, grid):
         if dom_j == 0:
             free_left = True
         else:
-            free_left = all([not grid[i, dom_j - 1] for i in
-                             range(start, end + 1)])
+            free_left = not all(grid[i, dom_j - 1] for i in
+                                range(start, end + 1))
```

Comment:
Should this not be changed to `any`?

Reply:
It should!

I'll also try to revert the changes fully in files where there are now only white space changes...